.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint:ox
+yarn lint-staged && yarn lint:ox && cargo fmt --all && git add .
```

Comment:
Maybe we could add lint:ox, cargo fmt to lint-stage config as well?
Also, lint-stage will do `git add` implicitly, as stated in https://github.com/lint-staged/lint-staged#:~:text=From%20v10.0.0%20onwards,in%20an%20error

Reply:
cargo fmt and lint:ox is fast, so I don't think it's not necessary to add it into lint-staged for performance reason.
Another consideration is that the rules of cargo fmt often change with the upgrade of Rust version. At this time, I hope to find out the changes in all files during pre-commit, rather than just the modified files.